jujugui/static/gui/src/app/assets/css/canvas/_service.scss

```diff
@@ -191,12 +190,20 @@
     }
 
     @-webkit-keyframes serviceBlockHaloSelected {
-      0% { opacity: 0; }
-      100% { opacity: .3; }
+      0% {
```

Comment:
I can see how this would be an improvement if you had multiple styles applied, but when there is just one, it's actually easier to glance with just the one 🤷‍♂️

Reply:
Yes, I would agree here. But, at the risk of sounding like a broken record, nobody has to like all of the things...